certification/internal/policy/operator/scorecard_check.go

```diff
@@ -42,6 +42,7 @@ func (p *scorecardCheck) getDataToValidate(bundleImage string, selector []string
 		Namespace:      namespace,
 		ServiceAccount: serviceAccount,
 		Verbose:        true,
+		WaitTime:       "120s",
```

Comment:
This just means it will wait up to 120s, not that it will wait 120s no matter what, right? If the former, I'm fine with this. If the latter, can we maybe start with 2x the default and see if that helps?

Reply:
This is from their `help` menu, not sure if it provides more clarity or not.
`-w, --wait-time duration       seconds to wait for tests to complete. Example: 35s (default 30s)`

Reply:
@bcrochet @acornett21 it is the former; it will wait up to 120s.